composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "CascadeEnergy/distributed-locking",
+  "description": "Base distributed locking library",
+  "minimum-stability": "stable",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Charles Berube",
+      "email": "charles.berube@cascadeenergy.com"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "Cascade\\": "src/"
```

Comment:
Could eliminate the `DistributedLocking` directory, by just making this:

```
"Cascade\\DistributedLocking\\": "src/"
```

It's all in that namespace anyhow, avoid directory digging.